vg/draw/canvas.go

```diff
@@ -279,14 +317,16 @@ func (c *Canvas) Y(y float64) vg.Length {
 // Crop returns a new Canvas corresponding to the receiver
 // area with the given number of inches added to the minimum
 // and maximum x and y values of the Canvas's Rectangle.
-func (c Canvas) Crop(minx, miny, maxx, maxy vg.Length) Canvas {
+// Note that cropping the right and top sides of the canvas
+// requires specifying negative values of xmaxcrop and ymaxcrop.
+func (c Canvas) Crop(xmincrop, ymincrop, xmaxcrop, ymaxcrop vg.Length) Canvas {
```

Comment:
Sorry, I was referring to the change to parameter names; I don't think suffixing them all with crop adds anything.